pkgs/development/libraries/fmt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, cmake, shared ? false }:
```

Comment:
We usually prefer shared libraries in nixpkgs.

Reply:
I would also rename it to `enableShared` otherwise it can easily collide, when somebody adds a package called `shared`.

Reply:
shared was renamed to enableShared and defaults to true.